main.go

```diff
@@ -33,7 +33,13 @@ func addRootCmd(cmd *cobra.Command) *cobra.Command {
 func contextualize(fn entity.HandlerFunction, panicFn entity.PanicFunction) entity.CobraFunction {
 	return func(cmd *cobra.Command, args []string) error {
 		ctx := context.Background()
+
 		defer func() {
+			// Let CLI fail if we're runnign the dev version
+			if !constants.IsDevVersion() {
+				return
+			}
```

Comment:
Hmmm, should we be doing this? At the bare minimum would be great to have a comment why

Reply:
Will add comment to this. Basically, this will let Go print the panic trace instead of us intercepting and showing the "report to railway" bug flow

Reply:
But, the logic is actually backwards, so I'll fix that 😆

Reply:
Also, here's a screenshot, for completeness
<img width="943" alt="image" src="https://user-images.githubusercontent.com/587576/121266798-fe66b100-c86f-11eb-9529-ed7b1fbc6235.png">